projects/lights_control/inc/lights_blinker.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+#include "event_queue.h"
+#include "soft_timer.h"
+
+// This module aids with the blinking functionality that some lights require such as signal lights,
```

Comment:
Blinking requires `soft_timers` and `event_queue` to be initialized. Please indicate this.